index.html

```diff
@@ -201,10 +201,10 @@ <h4>CANOPY</h4>
 
              </div>
 
-         </div>
+         </main>
```

Comment:
- I recommend that you don't forget to change your </main> tag to </section> to follow good HTML practices.